internal/store/publish.go

```
@@ -75,20 +77,42 @@ func PublishToRegistry(dst Publisher, ref string) (*PublishResult, error) {
+	}
+	drawerDir, err := config.DrawerDir(slug)
 	if err != nil {
 		return nil, err
 	}
+	buttonSpec := filepath.Join(btnDir, "button.json")
+	drawerSpec := filepath.Join(drawerDir, "drawer.json")
+	buttonExists := fileExists(buttonSpec)
+	drawerExists := fileExists(drawerSpec)
+	switch {
+	case buttonExists && drawerExists:
+		return nil, fmt.Errorf("ambiguous package %q: found both button and drawer. Rename one.", slug)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)